internal/hcsoci/hcsdoc_lcow.go

```diff
@@ -51,7 +51,22 @@ func createLCOWSpec(coi *createOptionsInternal) (*specs.Spec, error) {
 
 	// Clear unsupported features
 	if spec.Linux.Resources != nil {
-		spec.Linux.Resources.Devices = nil
+		if spec.Linux.Resources.Devices != nil {
+			// We dont know the host device list for LCOW pre-create. For
```

Comment:
I'm not sure I understand the logic here. Talk in-person tomorrow and explain please :)